AirSkyBoat/AirSkyBoat
# HNM/NM Tuning, Fixes Evasion for mobs, Dream Flower Addition,

**_I affirm:_**

## What does this pull request do?
- Added Evasion Total to !getstats
- Tuning for all HNMs/Wyrms, Sky Gods and Jailers
- Fixes Aspid and Tartaruga goingg into shell too early
- Added Dream Flower to mandy jug pet

Fixes: https://github.com/HorizonFFXI/HorizonXI-Issues/issues/459


## Steps to test these changes
- Fight all HNMs, Sky Gods and Jailers and die faster
- Fight Aspid and see it not go into its shell immediately 
- Call Beast for the mandy pet and see Dream Flower
